packages/elements/src/internals/machines/sign-in/verification.machine.ts

```diff
@@ -448,17 +448,6 @@ export const SignInFirstFactorMachine = SignInVerificationMachine.provide({
 
           break;
         }
-        case 'web3_coinbase_signature': {
-          const signature = fields.get('signature')?.value as string | undefined;
-          assertIsDefined(signature, 'Web3 Coinbase signature');
-
-          attemptParams = {
-            strategy,
-            signature,
-          } satisfies Web3Attempt;
-
-          break;
-        }
```

Comment:
Are we removing the elements support for Coinbase completely or are we replacing it with Coinbase Wallet ? Will this be done in a follow up PR ?

Reply:
In this PR we are removing the support for the deprecated Coinbase completely. There is a separate PR https://github.com/clerk/javascript/pull/4103 to handle the new Coinbase Wallet provider